site/lib/layouts/dash_layout.dart

```diff
@@ -0,0 +1,204 @@
+      link(
+        rel: 'icon',
+        href: '/assets/img/logo/dart-64.png',
+        attributes: {'sizes': '64x64'},
+      ),
+      link(
+        rel: 'apple-touch-icon',
+        href: '/assets/img/touch-icon-iphone.png',
+      ),
+      link(
+        rel: 'apple-touch-icon',
+        href: '/assets/img/touch-icon-ipad.png',
+        attributes: {'sizes': '152x152'},
+      ),
+      link(
+        rel: 'apple-touch-icon',
+        href: '/assets/img/touch-icon-iphone-retina.png',
+        attributes: {'sizes': '180x180'},
+      ),
+      link(
+        rel: 'apple-touch-icon',
+        href: '/assets/img/touch-icon-ipad-retina.png',
+        attributes: {'sizes': '167x167'},
+      ),
+      meta(name: 'twitter:card', content: 'summary'),
+      meta(name: 'twitter:site', content: '@dart_lang'),
+      meta(name: 'twitter:title', content: pageTitle),
+      meta(
+        name: 'twitter:description',
+        content: '${pageData['description']}',
+      ),
+
+      meta(attributes: {'property': 'og:title', 'content': pageTitle}),
+      meta(
+        attributes: {
+          'property': 'og:description',
+          'content': '${pageData['description']}',
+        },
+      ),
+      meta(attributes: {'property': 'og:url', 'content': page.path}),
+      meta(
+        attributes: {
+          'property': 'og:image',
+          'content': '/assets/img/logo/dart-logo-for-shares.png',
+        },
+      ),
+
+      link(rel: 'preconnect', href: 'https://fonts.googleapis.com'),
+      link(
+        rel: 'preconnect',
+        href: 'https://fonts.gstatic.com',
+        attributes: {'crossorigin': ''},
+      ),
+      link(
+        rel: 'stylesheet',
+        href:
+            'https://fonts.googleapis.com/css2?family=Google+Sans:wght@400;500;700&display=swap',
+      ),
+      link(
+        rel: 'stylesheet',
+        href:
+            'https://fonts.googleapis.com/css2?family=Google+Sans+Mono:wght@400;500;700&display=swap',
+      ),
+      link(
+        rel: 'stylesheet',
+        href:
+            'https://fonts.googleapis.com/css2?family=Google+Sans+Text:wght@400;500;700&display=swap',
+      ),
+      link(
+        rel: 'stylesheet',
+        href:
+            'https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0..1,0',
+      ),
+      link(rel: 'stylesheet', href: '/assets/css/main.css'),
+
+      script(src: '/assets/js/tabs.js'),
+      script(src: '/assets/js/main.js'),
+      if (pageData['js'] case final List<Object?> jsList)
+        for (final js in jsList)
+          if (js case {'url': final String jsUrl, 'defer': final Object? defer})
+            script(
+              src: jsUrl,
+              attributes: {if (defer == 'true' || defer == true) 'defer': ''},
+            ),
+      script(
+        src:
+            'https://cdn.jsdelivr.net/npm/@justinribeiro/lite-youtube@1.8.2/lite-youtube.js',
```

Comment:
Is this a new dependency, or something we've been using for a while?

Reply:
We've been using this awhile as this helped us significantly lower page load time for pages with embedded YouTube videos. I've gone ahead and added some inline comments explaining the inclusion.

Thanks for considering this!